include/manifold/manifold.h

```diff
@@ -13,12 +13,12 @@
 // limitations under the License.
 
 #pragma once
+#include <fstream>
 #include <functional>
-#include <memory>
-
-#ifdef MANIFOLD_EXPORT
+#include <iomanip>
 #include <iostream>
-#endif
+#include <memory>
+#include <string>
```

Comment:
So, we were intentionally keeping things like `<string>` from being included in the base library because they have a tendency to bloat WASM builds considerably. I haven't tested this recently - can you show us the difference in WASM build size before and after this change?

Reply:
Um.  I'm not set up to build WASM, so that'd take some time for me to figure out.  What about using a different guard than MANIFOLD_EXPORT?  What I'd like to do is have this I/O capability without needing to rope in assimp - or, if we want to use MANIFOLD_EXPORT for this, maybe switch the MeshIO guard to MANIFOLD_ASSIMP instead?

Reply:
I can see wanting to keep <string> out of the base build if size is an issue - even if it happens to be not-large this go around (I'm guessing that's pretty unlikely) there's no guarantee going forward it would stay that way.  I would like to have the ability to read and write the OBJs both in debug and release build configs and not need to patch to avoid roping in assimp, but beyond that I've not got any really strong preferences.

Reply:
One of my potential use cases for this ability is to be able to automatically dump out reproduction inputs if/when a problem case arises - there's no telling if/when that will happen, so ideally I'd like to not have to worry about recompiling Manifold if I run into an issue.  It's possible I'm misinterpreting what MANIFOLD_DEBUG is for - I had the impression it was to toggle on code that could potentially have performance implications if left on all the time, but maybe there's no real penalty for enabling it beyond library size?

Reply:
MANIFOLD_DEBUG should just be about removing code from the release build - it shouldn't actually run anything extra on its own, but just allow things like the verbose flag and extra checks flag to do something when also enabled. I should update our docs to make that more clear. I just looked through the code and found two places that's not currently true, so we'll fix those. 

Meanwhile, yes I think this is a perfect function to have guarded by `MANIFOLD_DEBUG`. Since it'll be public API, I guess just guard the implementation and on the `#else` side just return an error Manifold or a boolean flag.

Reply:
The README indicates the MANIFOLD_DEBUG option incurs about 20% runtime overhead?

Reply:
I'm hoping that's really when `extraChecks` is enabled. If not, we'll make it that way.